src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -69,5 +84,10 @@ object SparkBigQueryOptions {
       fallback: Option[String] = None): Option[String] = {
     options.get(name.toLowerCase).orElse(fallback)
   }
+
+  private def getAnyOption(globalOptions: Map[String, String],
+                           options: Map[String, String],
+                           name: String): Option[String] =
+    options.get(name.toLowerCase).orElse(globalOptions.get(name.toLowerCase))
```

Comment:
why is toLowerCase necessary here? Shouldn't `name` be case sensitive?

Reply:
Good point - base64 is case sensitive

Reply:
option names: credentials and credentialsFile should be case sensitive.

Reply:
done.